cpp/ql/lib/experimental/Quantum/OpenSSL/AlgorithmInstances/KnownAlgorithmConstants.qll

```diff
@@ -202,6 +253,11 @@ predicate defaultAliases(string target, string alias) {
   alias = "ssl3-sha1" and target = "sha1"
 }
 
+predicate tbd(string normalized, string algType) {
```

Comment:
Normalization to key size / alg family type should occur as it does in our JCA modelling and not via intermediate string types: map it to the 'fixed' internal family type as defined as Model.qll, with key size or algorithm variants defined separately (in this code, AES-128 and SHA-256 are two examples that stick out).